File: Modul6/Python/PRAK602-2310817310012-MUHAMMAD_AZWIN_HAKIM.py
def hitung_zetsu_putih(ruangan, zetsu):
    hasil = []
    for i in range(ruangan):
        hasil.append(zetsu[i] * (i + 1))
    return hasil

ruangan = int(input())
zetsu = list(map(int, input().split()))

print()

hasil_hitung = hitung_zetsu_putih(ruangan, zetsu)
print(*hasil_hitung)

File: Modul6/C/PRAK602-2310817310012-MUHAMMAD_AZWIN_HAKIM.c
#include <stdio.h>

int main() {
    int Ruangan;
    scanf("%d", &Ruangan);

    int Zetsu[Ruangan];
    for (int i = 0; i < Ruangan; i++){
        scanf("%d", &Zetsu[i]);
    }printf("\n");
    
    for (int i = 0; i < Ruangan; i++){
        printf("%d ",  Zetsu[i] * (i + 1));
    }
    return 0;
}